src/bm_sim/switch.cpp

```diff
@@ -252,8 +252,9 @@ SwitchWContexts::init_from_options_parser(
   if (parser.console_logging)
     Logger::set_logger_console();
 
-  if (parser.file_logger != "")
-    Logger::set_logger_file(parser.file_logger, parser.log_flush);
+  if (parser.file_logger != ""){
```

Comment:
```suggestion
  if (parser.file_logger != "") {
```

isn't this detected by the style checker?